### PR TITLE
[Deploy command — 06] Deployment guide

### PR DESCRIPTION
**Pull request by kasperski95**

### CODE REVIEW

File: website/docs/tutorials/guides/deployment.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: Deployment
+---
+
+# Deployment
+
+Protostar `>=0.2.1` supports deploying smart contracts to a given network with the `protostar deploy` command. It has similar interface to `starknet deploy` command. [Read the CLI reference for the deploy command](/docs/cli-reference#deploy) to learn more about all supported arguments.
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deployment.md:7]: I think the information `>=0.2.1` is unnecesary, it's place is in changelog, not in a guide.

Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deployment.md:7]:
```suggestion
Protostar `>=0.2.1` supports deploying smart contracts to a given network with the `protostar deploy` command. It has a similar interface to the `starknet deploy` command. [Read the CLI reference for the deploy command](/docs/cli-reference#deploy) to learn more about all supported arguments.
```

File: website/docs/tutorials/guides/deployment.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Example — deploying the default contract
+1. Create new project 
+   ```bash
+   $ protostar init
+   ```
+2. Build the project
+   ```bash
+   $ protostar build
+   ```
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deployment.md:17]: Should we include it in this section or linik init tutorial? Im not sure, your call

File: website/docs/tutorials/guides/deployment.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    ```shell title="Deployment output"
+    [INFO] Deploy transaction was sent.
+    Contract address: 0x06a5ea9e42c921bd58e24b8da9d1fc91a488df0700b173f1c6bb0e453f68afec
+    Transaction hash: 0x1cbba90ba0d1fbfba09b1f7a0f987134dd9a02a845ca89244b3272374d37ede
+
+    https://goerli.voyager.online/contract/0x06a5ea9e42c921bd58e24b8da9d1fc91a488df0700b173f1c6bb0e453f68afec
+    ```
+
+## Using [configuration profiles](/docs/tutorials/project-initialization#configuration-profiles)
+Configuration profiles allow you to easily reuse configuration for devnet, testnet, and mainnet networks. You can do it by defining the following configuration profiles in the `protostar.toml` as demonstrated on the snippet below.
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deployment.md:31]: I think configuration profiles are more general and should have their own guide, it suggest they can't be used for other commands.

Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deployment.md:31]:
```suggestion
Configuration profiles allow you to easily reuse configuration for devnet, testnet, and mainnet networks. You can definine sample configuration profiles in the `protostar.toml` as demonstrated on the snippet below.
```

**Review by MaksymilianDemitraszek — CHANGES_REQUESTED** [review]
LGTM, some comments. 

Asid, I think we should provide a section about using a configuration file to configure command arguments, it may be not obvious to every user that each flag can be configured in a config file (maybe not in this PR though).

File: website/docs/tutorials/guides/deployment.md (hunk 3 above)
Comment (MaksymilianDemitraszek) [reply website/docs/tutorials/guides/deployment.md]: Ok, i missed that, can we link it here?
> If you want to know more about configuration profiles go HERE

or something similar

Comment (MaksymilianDemitraszek) [reply website/docs/tutorials/guides/deployment.md]: Nevermind, we already do ❤️

File: website/docs/tutorials/guides/deploying.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## Example — deploying the default contract
+After [compiling your project](/docs/tutorials/guides/compiling), you can deploy the contract in the following way.
+
+```
+$ protostar deploy ./build/main.json --network alpha-goerli
+```
+  
+ ```shell title="Deployment output"
+[INFO] Deploy transaction was sent.
+Contract address: 0x06a5ea9e42c921bd58e24b8da9d1fc91a488df0700b173f1c6bb0e453f68afec
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deploying.md:18]: Maybe we should put mock values here 😛 ? Idk what is deployed there though just raising it as a random thought

File: website/docs/tutorials/guides/deploying.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: Deploying (1 min)
+---
+
+# Deploying
+
+Protostar supports deploying smart contracts to a given network with the `protostar deploy` command. It has a similar interface to the `starknet deploy` command. [Read the CLI reference for the deploy command](/docs/cli-reference#deploy) to learn more about all supported arguments.
+
+## Example — deploying the default contract
+After [compiling your project](/docs/tutorials/guides/compiling), you can deploy the contract in the following way.
```
Comment (MaksymilianDemitraszek) [inline website/docs/tutorials/guides/deploying.md:10]:
```suggestion
After [compiling your contract](/docs/tutorials/guides/compiling), you can deploy the contract in the following way.
```
A slightly better wording i think

**Review by MaksymilianDemitraszek — APPROVED** [review]

File: website/docs/tutorials/guides/deploying.md (hunk 4 above)
Comment (kasperski95) [reply website/docs/tutorials/guides/deploying.md:18]: IMO it doesn't  matter

**Pull request merged**